unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -61,7 +61,7 @@ function cert_generateAdmincertificate() {
     eval "openssl pkcs8 -inform PEM -outform PEM -in /tmp/wazuh-certificates/admin-key-temp.pem -topk8 -nocrypt -v1 PBE-SHA1-3DES -out /tmp/wazuh-certificates/admin-key.pem ${debug}"
     eval "openssl req -new -key /tmp/wazuh-certificates/admin-key.pem -out /tmp/wazuh-certificates/admin.csr -batch -subj '/C=US/L=California/O=Wazuh/OU=Wazuh/CN=admin' ${debug}"
     eval "openssl x509 -days 3650 -req -in /tmp/wazuh-certificates/admin.csr -CA /tmp/wazuh-certificates/root-ca.pem -CAkey /tmp/wazuh-certificates/root-ca.key -CAcreateserial -sha256 -out /tmp/wazuh-certificates/admin.pem ${debug}"
-
+    cert_setpermisions
```

Comment:
I would change the function call to this place:
https://github.com/wazuh/wazuh-packages/blob/2e1630dd0290742ef43149b5dae2ec90c25d6711/unattended_installer/cert_tool/certMain.sh#L159-L164

Reply:
Done [ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41](https://github.com/wazuh/wazuh-packages/pull/1415/commits/ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41)